coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 6tunIUhG8GG8WNmsdDH6MnGhpLMjU5M5y
```

Comment:
There's should be a line at the file, also if this is not a public token find a way to grab it from an `.env` var